src/hooks/useTypeahead.ts

```diff
@@ -25,15 +25,13 @@ const useTypeaheadQuery = (
   query: string,
   onTransform?: <Data>(data: Data) => any
 ) => {
-  const searchQuery = useQuery(
-    ["typeahead"],
-    () => fetchSearchResults(url, queryParamKey, query),
-    {
-      enabled: Boolean(query),
-      select: onTransform,
-      retry: 1,
-    }
-  );
+  const searchQuery = useQuery({
+    queryKey: ["typeahead"],
+    queryFn: () => fetchSearchResults(url, queryParamKey, query),
+    enabled: Boolean(query),
+    select: onTransform,
+    retry: 1,
```

Comment:
Has this been tested? Why was this put in to an options object in the first place? Is this due to an updated library?

Reply:
Hm, its not used according to Github search.

We should audit and delete all unused components (in a new ticket)